app/models/users.py

```diff
@@ -0,0 +1,20 @@
+from typing import Union
+from sqlmodel import Field, SQLModel
+
+class Token(SQLModel):
+    """JSON Web Token (JWT) provided to the user after authentication."""
+    access_token: str
+    token_type: str
+
+class TokenData(SQLModel):
+    """TokenData links the JWT token to the username"""
+    username: Union[str, None] = None
+
+class Users(SQLModel, table=True):
+    """Users are required to be authenticated to use some functionality of the API.
+    Disabled users are unable to authenticate to receive a token."""
```

Comment:
When writing DocStrings the final """ should be on a new line as per [https://peps.python.org/pep-0257/#multi-line-docstrings](https://peps.python.org/pep-0257/#multi-line-docstrings)